src/note.cpp

```diff
@@ -398,6 +401,53 @@ wchar_t Note::GetMensuralSmuflNoteHead()
     return code;
 }
 
+wchar_t Note::GetNoteheadGlyph(const int duration) const 
+{
+    static std::map<std::string, wchar_t> additionalNoteheadSymbols
+        = { { "noteheadDiamondBlackWide", SMUFL_E0DC_noteheadDiamondBlackWide },
+            { "noteheadDiamondWhiteWide", SMUFL_E0DE_noteheadDiamondWhiteWide },             
+            { "noteheadNull", SMUFL_E0A5_noteheadNull } };
```

Comment:
Are these missing in MEI? If yes, then it would be good to see if they should be added to MEI.

Reply:
They are missing in MEI, yes. It would be good if noteheadNull were to be added, but other too feel more like alternatives to diamond shape that is already in the MEI. Albeit slightly wider, I'm not sure if that warrants a new change in MEI.

Reply:
But that feels more like enhancement that can be addressed separately when (or if) MEI adds them.

Reply:
OK, thanks.